app/models/order.rb

```diff
@@ -1,11 +1,11 @@
 class Order < ApplicationRecord
   belongs_to :kit
   belongs_to :user
+  has_one :address, as: :addressable
```

Comment:
I realize the ticket said this, but I think we perhaps realized that a `User` should own an `Address` instead on Wednesday. 

I am thinking that it is useful for an `Order` to store an explicit reference to an `Address`, however, for booking keeping purposes: What if I place an order, change my address after it ships, and then am upset that my order didn't arrive at the new address (which our DB now has no record of being associated with that Order, either)?

Perhaps we need to add `address_id` to `Order`, after all. Interested in other's thoughts

Reply:
sounds great!

Reply:
It should be fine for future bookkeeping purposes. Although, if most of the users are teachers, their orders would be mailed directly to their schools, right?

Reply:
@olesiamironenko can you add a migration to this PR to `add_reference` to `address` on `orders` table? https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference

Reply:
db/migrate/20241218050622_add_address_id_to_orders.rb